packages/core/src/spec/lib/jdk/KeyToolSpec.ts

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import {KeyTool, CreateKeyOptions, KeyOptions} from '../../../lib/jdk/KeyTool';
+import {Config} from '../../../lib/Config';
+import util = require('../../../lib/util');
+import {JdkHelper} from '../../../lib/jdk/JdkHelper';
+import * as fs from 'fs';
+
+/* eslint-disable max-len */
+const LIST_OUTPUT = `Alias name: key0
+Creation date: 28 Jan 2019
+Entry type: PrivateKeyEntry
+Certificate chain length: 1
+Certificate[1]:
+Owner: CN=Test Test, OU=Test, O=Test, L=London, ST=London, C=GB
+Issuer: CN=Test Test, OU=Test, O=Test, L=London, ST=London, C=GB
+Serial number: ea67d3d
+Valid from: Mon Jan 28 14:58:00 GMT 2019 until: Fri Jan 22 14:58:00 GMT 2044
+Certificate fingerprints:
+   SHA1: 38:03:D6:95:91:7C:9C:EE:4A:A0:58:43:A7:43:A5:D2:76:52:EF:9B
```

Comment:
You could do:

    const SHA1_KEY = "....";
    const SHA256_KEY = "....";
    const LIST_OUTPUT = `
    ....
        SHA1: ${SHA1_KEY}
    ....`

So you don't have to repeat yourself in the test. Up to you though.

Reply:
done.